ee/tables/tablehelpers/posix_run_cmd_as_user.go

```diff
@@ -0,0 +1,66 @@
+//go:build !windows
```

Comment:
I think the convention is to make the OS the suffix, not the prefix -- `run_cmd_as_user_posix.go`. (I couldn't quickly find great docs on this, but it's mentioned [here](https://www.digitalocean.com/community/tutorials/building-go-applications-for-different-operating-systems-and-architectures#using-goos-and-goarch-filename-suffixes) or you can see e.g. [here](https://pkg.go.dev/golang.org/x/sys@v0.18.0/unix#section-sourcefiles).)

Reply:
Sounds good. I questioned how I should've named it and thinking a bit more I agree with the suffix.

Reply:
IIRC early go used the suffixes as implicit build tags, and that ended up being an antipattern. So now the convention is filename suffixes, and build tags.

Reply:
(This is how the `_test.go` files don't get built into the prod binaries)